src/System.Drawing.Primitives/src/System/Drawing/Size.cs

```diff
@@ -4,49 +4,39 @@
 
 using System.ComponentModel;
 using System.Numerics.Hashing;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
 
 namespace System.Drawing
 {
-    /**
-     * Represents a dimension in 2D coordinate space
-     */
     /// <summary>
-    ///    Represents the size of a rectangular region
-    ///    with an ordered pair of width and height.
+    /// Represents the size of a rectangular region with an ordered pair of width and height.
     /// </summary>
     [Serializable]
-    [System.Runtime.CompilerServices.TypeForwardedFrom("System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a")]
+    [ComVisible(true)]
```

Comment:
Do we understand the implications of marking these types as `ComVisible`? Is the public COM surface of these type compatible with the COM surface that there types had in .NET Framework?

We may want to have tests that exercise the COM surface of these types to make sure that they do actually work when used via COM.

cc @AaronRobinsonMSFT

Reply:
Our strategy in .NET Core was to minimize number of COM visible types because of they come with a lot  maintenance burden. We have marked the types as COM visible only when there was a customer asking for it. Is there a customer asking for these types to be marked as COM visible?

Reply:
> Is there a customer asking for these types to be marked as COM visible?

I would be very interested if there were.